youtube_dl/extractor/redtube.py

```diff
@@ -91,7 +91,7 @@ def _real_extract(self, url):
             r'<span[^>]+>ADDED ([^<]+)<',
             webpage, 'upload date', fatal=False))
         duration = int_or_none(self._search_regex(
-            r'videoDuration\s*:\s*(\d+)', webpage, 'duration', default=None))
+            '<meta.*property="og:video:duration".*content="(.*)".*\/>', webpage, 'duration', default=None))
```

Comment:
_og_search_property`.

Reply:
thx for your commitment to help out a newbie. I will read the code guidelines fully before I continue